webui/assets/style.css

```diff
@@ -295,4 +295,42 @@ td{
 
 .expand {
   text-align: center;
-}
\ No newline at end of file
+}
+
+.query h2 {
+  display: inline-block;
+}
+
+.query-tooltip {
+  position: relative;
+  display: inline;
+  color: #1db598;
+  margin-left: 5px;
+}
+
+.query-tooltip-content {
+  visibility: hidden;
+  background-color: rgb(250, 250, 250);
+  border: solid 1.5px #e4eff4;
+  color: #102445;
+  border-radius: 2px;
+  padding: 15px;
+  margin-bottom: 15px;
+
+  /* Position the tooltip */
+  position: absolute;
+  /*
```

Comment:
This was me trying to figure out how to move the tooltip without breaking everything else